### ISSUE TITLE
https://webauthn.io/ supports any device but this demo doesn't do that

### ISSUE THREAD

**Issue by Naggertooth** [fido-alliance/webauthn-demo#19]
Hi guys!

Is that possible to supple this demo to support any device as it is on your website?

Or may be you guys have another example or tutorial or something?

Sorry if i missed something ^^

**Comment by Perlover** [comment]
+1

Firefox 71.0 + node v12.13.1 + Linux Mint 19.2

Developers, your demo site works for me fine but this demo from localhost:3000 - nothing happens. I see login form, i do register -> fill form -> submit -> i see login page again -> submit name -> i see login page and so on.

In development tools in Net i see "304" GET responses for each request only... Everywhere...

**Comment by Naggertooth** [comment]
> +1
> 
> Firefox 71.0 + node v12.13.1 + Linux Mint 19.2
> 
> Developers, your demo site works for me fine but this demo from localhost:3000 - nothing happens. I see login form, i do register -> fill form -> submit -> i see login page again -> submit name -> i see login page and so on.
> 
> In development tools in Net i see "304" GET responses for each request only... Everywhere...

https://github.com/fido-alliance/webauthn-demo/issues/20#issuecomment-565383517

**Comment by Naggertooth** [comment]
The main issue is still not solved

**Comment by Perlover** [comment]
> The main issue is still not solved

Yes. Simple this code is not completely completed and completely inoperative: https://github.com/fido-alliance/webauthn-demo/issues/21

**Comment by Naggertooth** [comment]
> Yes. Simple this code is not completely completed and completely inoperative: #21

@Perlover 
Dude, I am not one of devs of this repo :)
I just really don't have time at all to solve it by myself

So please don't spam in my issue^^
If something does not supported by maintainers - it doesn't mean that it doesn't work
It works
Not completed product, but works!

And my issue is about any device
Not about workin
I got workin example from this repo btw (i didnt test it with 'fido-u2f' as it expected but I guess it works fine with it)

And if u read readme - there is completed demo at completed demo branch
Afterwards you can complete your code from master if it needs to be done

**Comment by Hexagon** [comment]
A working and improved version (with Windows hello and android support) available at http://github.com/hexagon/webauthn-skeleton
